messages/renderer/vi.json

```diff
@@ -274,7 +243,7 @@
   },
   "renderer.components.MapFilter.ObservationDialog.ObservationDialog.saveEditButton": {
     "description": "Save edit button",
-    "message": "Lưu"
+    "message": "Save"
```

Comment:
@gmaclennan why is it doing this?

Reply:
This is happening because this particular translation came from the merge of MapFilter, however the translations on CrowdIn have not been updated (this string is untranslated on CrowdIn). What likely happened is that the source English strings updated, CrowdIn read them from Github, then generated English defaults for the language, then replaced what is in the repo. The resolution is to upload the translations from messages/renderer/vi.json to CrowdIn manually, to make sure all the strings from that file are in CrowdIn.

Also, we should change the CrowdIn configuration to not include untranslated strings in language exports, because react-intl already falls back to the English version (the current CrowdIn behaviour is useful if using an i18n solution that does not have the English fallback strings). See digidem/mapeo-mobile#478 — I am waiting to merge v5.1 before doing this on Mobile because it changes a lot of files and currently we have translations across two branches with the release.

Reply:
OK Thanks @gmaclennan. Do you have time to do this? We should make a release of 5.4.0 with these translations included